torch_xla/csrc/tensor_methods.cpp

```diff
@@ -730,6 +730,12 @@ XLATensorPtr argmin(const XLATensorPtr& input) {
 XLATensorPtr as_strided(const XLATensorPtr& input, std::vector<int64_t> size,
                         std::vector<int64_t> stride,
                         c10::optional<int64_t> storage_offset) {
+  // See Note: [Disabling functionalization]
+  if (xla::sys_util::GetEnvBool("XLA_DISABLE_FUNCTIONALIZATION", false)) {
```

Comment:
I'm trying to figure out how you fix the test. Right now, which branch is being run, the if block or the else block?

Reply:
By default the flag is off, we have a dedicated run of test_operations.py with the flag on.